### ISSUE TITLE
Prefs: Allow changing orientation of non-expanded panels

### ISSUE THREAD

**Issue by cwendling** [mate-desktop/mate-panel#1213]
#### Expected behaviour
In a panel's preference window, be able to change the orientation of a non-expanded panel just like an expanded one.

#### Actual behaviour
The sensitivity of the *Orientation* settings depends on *expand* being turned off.

This limitation seems to exist because of *floating* panels, which cannot be described by these preferences.  Only non-expanded panels can be floating, hence the tie with the *expand* checkbox.

However, it feels unexpected not to be able to place the panel because of it expanding or not, especially as setting the orientation *before* turning expand off works fine.  This feels confusing.

IMO, at least if the panel is not floating it should be possible to change its orientation, even if it's not expanded.

#### Steps to reproduce the behaviour
Open a panel's preferences, and uncheck *expand*.

#### MATE general version
1.25

**Comment by zhuyaliang** [comment]
I'm at ```data/org.mate.panel.toplevel.gschema.xml.in ``` file, I hope it is useful for you
```
<description>The orientation of the panel. Possible values are "top", "bottom", "left", "right". In expanded mode the key specifies which screen edge the panel is on. In un-expanded mode the difference between "top" and "bottom" is less important - both indicate that this is a horizontal panel - but still give a useful hint as to how some panel objects should behave. For example, on a "top" panel a menu button will pop up its menu below the panel, whereas on a "bottom" panel the menu will be popped up above the panel.</description>
```

**Comment by cwendling** [comment]
Yeah I figured this when looking at the source, but it's really not obvious from a user POV, as this is *not* displayed anywhere in the UI. Moreover as the *orientation* combo becomes insensitive when un-expanding, it's not possible to change it when in non-expanded mode -- although it still affects the panel's actual orientation (and even position if changed before un-expanding).

All this is still manageable with tuning the gsettings when you know how, but it's not what I'd call user friendly.
